packages/adyen-salesforce-pwa/lib/api/middleware/request-context.js

```diff
@@ -35,7 +34,10 @@ export async function prepareRequestContext(req, res, next) {
     }
 
     try {
-        const basket = await getBasket(authorization, basketid, customerid)
+        let basket = null
```

Comment:
Making basket optional in the request-context could create issues with functions within BasketService. As operations in BasketService would expect adyenContext to have basket.
It would be good to use a different request-context for this usecase like maybe pdp-request-context

Reply:
This is done so the payment methods call can succeed without a basket